framework/sandstone.cpp

```diff
@@ -1650,6 +1648,10 @@ static void wait_for_children(ChildrenList &children, int *tc, const struct test
         children.results[idx] = test_result_from_exit_code(result);
         children.results[idx].endtime = MonotonicTimePoint::clock::now();
         --children_left;
+
+        // replace the handle with a valid, dummy, non-signalled handle
+        CloseHandle(handles[idx]);
+        children.handles[idx] = intptr_t(CreateEvent(nullptr, true, false, nullptr));
```

Comment:
[MSDN](https://learn.microsoft.com/en-us/windows/win32/api/synchapi/nf-synchapi-waitformultipleobjects) say that lpHandles may not contain multiple copies of the same handle. Maybe we can just create a single, "global" (static) null handle and reuse it?